<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>btTrack</class>
 <widget class="QWidget" name="btTrack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Track Logger</string>
  </property>
  <widget class="QGraphicsView" name="imageView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>721</width>
     <height>480</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="saveTrackButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>490</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadTrackButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>490</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearTrackButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>490</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="randomizeTrackButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Randomize</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noiseButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>490</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Noise</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveNoiseTrackButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>490</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save Noise</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>494</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Noise:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="simulateHButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>490</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SimulateH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="simulateAButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>490</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SimulateA</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
